packages/core/admin/admin/src/pages/SettingsPage/index.js

```diff
@@ -79,5 +68,5 @@ function SettingsPage() {
   );
 }
 
-export default memo(SettingsPage);
-export { SettingsPage };
+// Required for React.lazy
+export default SettingsPage;
```

Comment:
is it?

Reply:
No, it is fake news and I have fact-checked + rebased.